go.mod

```diff
@@ -1,10 +1,16 @@
 module github.com/fitzoh/jamie_chicken
 
+// 1.16rc1
+go 1.16
```

Comment:
Is go 1.16 required?
This is currently deployed via google cloud functions, looks like it maxes out at 1.13: https://cloud.google.com/functions/docs/concepts/exec

Reply:
nah, `go mod tidy` added it, will clean up later